build.gradle

```diff
@@ -129,9 +133,6 @@ publishing {
       // avoid infinite CircleCI builds triggered by release commits
       preCommitText = '[skip ci] '
       buildTasks = ['assemble']
-      git {
-        requireBranch = 'main'
```

Comment:
Default is already `main`
https://github.com/researchgate/gradle-release/blob/main/README.md#configuration
Also, the newest version for the plugin fails as that property is now final.